### DISCUSSION TITLE
Will homebrew work if I install homebrew by untarring to /opt/homebrew (writable)

### DISCUSSION THREAD

**Discussion by ywwry66** [Homebrew/discussions#5581] [Getting started] [answered]
### Output of `brew config`

```shell
NA
```

### Output of `brew doctor`

```shell
NA
```

### Description of issue

Basically I want to install homebrew on my work mac where I don't have sudo access. I think it might be easier to convince our IT staff to just create a writabe /opt/homebrew folder than letting them inspect every sudo usage in the homebrew installation script. So my question is will homebrew work normally if I just untar to the standard homebrew prefix for which I do have the write permission?

**Comment by ZhongRuoyu** [comment, marked as answer]
It should work fine. You can follow the steps in https://docs.brew.sh/Installation#untar-anywhere-unsupported, and if `brew doctor` doesn't complain, then you're good.

**Reply by ywwry66** [reply]
Thanks. I wil give it a try and report back.

**Reply by ywwry66** [reply]
Mission failed. No folder allowed for me under `/opt`. I am closing this discussion.
